e2e/assertions/compareScreenshot.js

```diff
@@ -57,11 +88,23 @@ exports.assertion = function(componentName, fileName) {
   }
 
   this.pass = function pass(value) {
-    const pass = value <= this.expected
+    let pass = value <= this.expected
 
-    this.message = `Screenshots ${
-      pass ? 'Matched' : 'Match Failed'
-    } for ${fileName} with a tolerance of ${this.expected}%, actual was ${value}%.`
+    if (
+      (process.env.UPDATE_SCREENSHOTS === 'false' &&
+        process.env.UPDATE_ALL_SCREENSHOTS === 'false') ||
+      pass
+    ) {
+      this.message = `Screenshots ${
+        pass ? 'Matched' : 'Match Failed'
+      } for ${fileName} with a tolerance of ${this.expected}%, actual was ${value}%.`
+    } else if (process.env.UPDATE_SCREENSHOTS === 'true') {
+      this.message = 'User was prompted to update screenshot.'
+      pass = true
```

Comment:
What if you were prompted and didn't respond with a "yes"?

Reply:
My initial idea for that is since the user actively approved a failing screenshot by choosing not to update it, it would count as a "pass". Although I was probably overthinking what one would consider a pass, so I fixed this in a commit I'm about to push.